plugins/embed-optimizer/hooks.php

```diff
@@ -63,7 +63,7 @@ function embed_optimizer_filter_oembed_html( string $html ): string {
 	if ( 1 === $script_count && ! $has_inline_script && $html_processor->has_bookmark( 'script' ) ) {
 		add_action( 'wp_footer', 'embed_optimizer_lazy_load_scripts' );
 		if ( $html_processor->seek( 'script' ) ) {
-			if ( $html_processor->get_attribute( 'type' ) ) {
+			if ( is_string( $html_processor->get_attribute( 'type' ) ) ) {
```

Comment:
In this case, I think the code should stay as-is.  Since `get_attribute` can return `true` in the case of a boolean attribute, we wouldn't want to try passing a `true` to `set_attrbute()`. I believe PHPStan would complain about that anyway.